docs/run.rst

```diff
@@ -143,3 +150,21 @@ Here is an example of a valid stage-in manifest file:
 ``/scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat``
 ``/home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf``
 ``"/home/users/me/file with space.dat" "/unifyfs/file with space.dat"``
+
+-----------------------------------------------
+  Stand-alone file stage application
+-----------------------------------------------
+The manifest subsystem can also be used during the job by
+invoking stand-alone transfer executable.  The command to use is
+"unify-static".  It's used similarly to the unix
```

Comment:
did you mean `unifyfs-transfer-static`?

Reply:
I guess the name of the example program is actually `transfer-static`

Reply:
Thank you.  Yes.  I was looking at the file "transfer-static" on catalyst, and my fingers typed "unify-static".  :-/  Oops.  I'll fix it.

Reply:
Fixed.